modules/installation-user-infra-machines-advanced-customizing-live.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+[id="installation-user-infra-machines-advanced-customizing-live-{boot}_{context}"]
+= Customizing a live {op-system} {boot-media}
+You can customize a live {op-system} {boot-media} directly with the
+ifeval::["{boot-media}" == "ISO image"]
+`coreos-installer iso customize`
+endif::[]
+ifeval::["{boot-media}" == "PXE environment"]
+`coreos-installer pxe customize`
+endif::[]
+subcommand. When you boot the {boot-media}, the customizations are applied automatically.
+
+You can use this feature to configure the {boot-media} to automatically install {op-system}.
+
+.Procedure
+
+. Download the `coreos-installer` binary from the link:https://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/latest/[`coreos-installer` image mirror] page.
+
+ifeval::["{boot-media}" == "ISO image"]
+. Retrieve the {op-system} ISO image from the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/[{op-system} image mirror] page and the Ignition config file, then run the following command to inject the Ignition config directly into the ISO image:
++
+[source,terminal]
+----
+$ coreos-installer iso customize rhcos-<version>-live.x86_64.iso \
+    --dest-ignition bootstrap.ign \ <1>
+    --dest-device /dev/sda <2>
+----
+endif::[]
+
+ifeval::["{boot-media}" == "PXE environment"]
+. Retrieve the {op-system} `kernel`, `initramfs` and `rootfs` files from the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/[{op-system} image mirror] page and the Ignition config file, then run the following command to create a new `initramfs` file that contains the customizations from your Ignition config:
++
+[source,terminal]
+----
+$ coreos-installer pxe customize rhcos-<version>-live-initramfs.x86_64.img \
+    --dest-ignition bootstrap.ign \ <1>
+    --dest-device /dev/sda \ <2>
+    -o rhcos-<version>-custom-initramfs.x86_64.img
+----
+endif::[]
+<1> The Ignition config file that is generated from `openshift-installer`.
+<2> When this option is specified, the {boot-media} automatically runs an install. Otherwise, the image remains configured for installing but will not do so automatically unless the `coreos.inst.install_dev` kernel argument is specified.
```

Comment:
Use present tense and add comma: "...configured for installing, but does not..."

Reply:
Thanks, fixed